README.md

```diff
@@ -9,55 +9,108 @@ http://52.24.131.202:8999/
 
 **NPM:** https://www.npmjs.com/package/gusto-stratera
 
+1)
+Assigned one leader, gave that person authority and responsibility, and held that person accountable for the quality of the prototype submitted.
 
-## Dependencies
-This is a node.js application which requires you to have node installed. To install node, please visit ```https://nodejs.org/download/``` and download and run the latest version of node.
+[See email](DevelopmentPoolEvidence/team/TechnicalLeadEmail.JPG)
 
-Alternatively, it's available for installation through a number of package managers, for example:
+2)
```

Comment:
Would like to get feedback from Vencore or need to debate.  I propose we do as we are saying and have John in both rolls with oversight by me and Marianne, Marianne as reviewer for submission purposes in this case, and me as technical (seeing as it is a prototype).  Regardless, Marianne, Steve and I will discuss in the morning and come up with a final answer by noon.